productivity-suite/piercing-library/src/message-bus/get-bus.ts

```diff
@@ -21,9 +25,16 @@ export function getBus(element?: Element): MessageBus {
   if (hasMessageBus(root)) {
     return root[messageBusProp];
   }
-  throw new Error("No global message bus defined!");
+  console.warn("No global message bus defined - creating an empty one.");
```

Comment:
I am not completely sure about the warn, if you run a fragment in isolation this is what you actually want right? so always displaying the warning anyways feels a bit off to me 😕

Reply:
If you are running the app in isolation then you are going to be getting placeholders instead of real fragments, which seems to me not a production scenario.

Reply:
yes but you'd still get the warning during development right?

Reply:
That's right! And I think that is a good thing. IMO.

Reply:
ah ok 🙂👍